liquidator/src/settings.js

```diff
@@ -0,0 +1,38 @@
+const EMP_ADDRESS = process.env.EMP_ADDRESS;
```

Comment:
having this in a separate file is pretty sweet! Quick question just so I understand: you don't validate that some of these parameters exist. In the previous version of this script if `EMP_ADDRESS` did not exist the script would exit early and tell the user with a thrown error that Winston would catch. How does this validation work now?

Reply:
@chrismaree there aren't any functions here so there's no way to validate anything. Perhaps we should wrap a bunch of these setters in an `init()` method that takes care of validations

Reply:
This was originally done because some of the tests dont' export `EMP_ADDRESS` by default, meaning the tests will fail.

I like what @nicholaspai said. Perhaps it'll be better to abstract this out into a function so we can inject the environment variable during the tests?

Reply:
sounds good to me! There needs to be some validation for these, especially the required fields.